recipes/openscenegraph/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.1.2)
+project(test_package)
+
+include(${PROJECT_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(OpenSceneGraph REQUIRED)
+
+include_directories(${OPENSCENEGRAPH_INCLUDE_DIRS})
```

Comment:
Maybe not for this PR, but could we use the targets generated by cmake_find_package generator instead of these global variables?

Reply:
AFAIK OpenSceneGraph doesn't have official CMake targets. I like to see official CMake usage of a library in test_package.

Reply:
I agree with @SpaceIm -- it is better for the test_package to test how most consumers will use the package, which in this case is global variables.